app/commands/v2/publish.rb

```diff
@@ -154,7 +154,7 @@ def clear_published_items_of_same_locale_and_base_path
       def set_public_updated_at
         return if edition.public_updated_at.present?
 
-        if update_type == "major"
+        if update_type == "major" || previous_item.blank?
```

Comment:
@thomasleese The other place that `update_type == "major"` is used in the publish function is to check whether to delete the change log. I think for self/auto publishers it's ok not to keep the change log.